### ISSUE TITLE
Reverse Distillation — Benchmark using MVTec training and testing

### ISSUE THREAD

**Issue by Tiff223** [Tiff223/Leather_inspection#18]
## Purpose

Establish the Reverse Distillation reference benchmark using MVTec AD leather for both fitting and evaluation.

This issue is one track of the model's three-track leather experiment. Benchmark, custom-trained, and MVTec-to-custom transfer results must remain separate and must never be averaged.

## Model-specific recording requirements

Record pretrained encoder/backbone, feature layers, anomaly-map combination mode, input size, epochs, optimizer settings, device, and all threshold/post-processing values.

## Pipeline preparation

Before execution:

- [ ] Create the MVTec and custom configuration files in the appropriate `configs/mvtec/` and `configs/custom_leather/` folders
- [ ] Implement and verify every required evaluator and report adapter
- [ ] Register `reverse_distillation` in `pipeline/leather_pipeline.json`
- [ ] Confirm every stage is `READY`; do not bypass a `NOT WIRED` stage
- [ ] Run dataset preflight and confirm no cross-split byte duplicates
- [ ] Preview the full master pipeline

Master pipeline preview:

```bash
python pythons/leather_three_track_pipeline.py plan --model reverse_distillation \
  --track all --run-id reverse_distillation_three_track_run01
```

Master pipeline execution:

```bash
python pythons/leather_three_track_pipeline.py run --model reverse_distillation \
  --track all --run-id reverse_distillation_three_track_run01 --execute
```

If execution stops, fix the cause and repeat the exact command with the same master run ID. Completed stages must be skipped and preserved.

## Dataset protocol

- Training: MVTec AD leather `train/good`
- Validation/calibration: document the exact Anomalib split convention
- Evaluation: standard MVTec AD leather test split
- Custom leather images used: 0
- Preserve the fitted checkpoint and every post-processing value required for the later frozen transfer track

## Track checklist

- [ ] Preview and execute `benchmark_fit`
- [ ] Verify checkpoint, master/child ledgers, configuration snapshot, and SHA-256 dataset manifest
- [ ] Preview and execute `benchmark_evaluate`
- [ ] Record image AUROC, AP, F1, accuracy, precision, recall, specificity, confusion matrix, runtime, and pixel metrics
- [ ] Record every image-level mistake and anomaly score
- [ ] Preview and execute `benchmark_report`
- [ ] Generate and integrity-check the DOCX report
- [ ] Execute the benchmark `update_comparison`
- [ ] Verify the result appears only in the MVTec benchmark section

## Artifacts and results

- Master run ID: `reverse_distillation_three_track_run01`
- Configuration:
- Checkpoint:
- Dataset manifest:
- Stage ledgers:
- Evaluation summary:
- Per-image scores:
- Report:
- Comparison files:

## Limitations

State threshold provenance clearly. If validation and test reuse the same MVTec images, identify threshold-dependent metrics as non-independent. Keep this result out of the custom-trained ranking.

## Conclusion

Not completed yet. Close only after the local artifacts and comparison placement have been reviewed.
